test/e2e/apimachinery/watch.go

```diff
@@ -314,6 +316,49 @@ var _ = SIGDescribe("Watchers", func() {
 		expectEvent(testWatch, watch.Modified, testConfigMapThirdUpdate)
 		expectEvent(testWatch, watch.Deleted, nil)
 	})
+
+	/*
+						    Testname: watch-consistency
```

Comment:
Something
      weird
    happened
   to
  this comment

Reply:
gah! fixed